configs/_base_/datasets/coco_detection.py

```diff
@@ -29,6 +29,7 @@
         ])
 ]
 data = dict(
+    # default value for all dataloader
     samples_per_gpu=2,
```

Comment:
We should deprecate this usage, so we can clean lines 32&33 in our own configs. Users will meet warning if they do not change their configs.

Reply:
In our and mmcls‘s design， the value in the data is the default value of all data_loader

Reply:
leaving them here will cause ambiguity. Users will be confused about where should they set samples_per_gpu

Reply:
I will add comments for the behavior